src/components/Popover/Popover.stories.tsx

```diff
@@ -1,34 +1,144 @@
 import { StoryObj, Meta } from '@storybook/react';
+import { within } from '@storybook/testing-library';
+import isChromatic from 'chromatic/isChromatic';
 import React from 'react';
-
-import { PopoverExample } from './PopoverExample';
+import { Popover } from './Popover';
+import Button from '../Button';
 
 export default {
   title: 'Organisms/Interactive/Popover',
-  component: PopoverExample,
+  component: Popover,
   parameters: {
     layout: 'centered',
   },
+  args: {
+    children: (
+      <>
+        <Popover.Button as={Button} data-testid="popover-trigger-button">
+          Open Popover
+        </Popover.Button>
+        <Popover.Content data-testid="popover-content">
+          <div style={{ width: '250px' }}>
+            My popover content Lorem ipsum dolor sit amet, consectetur
+            adipiscing elit, sed do eiusmod
+          </div>
+        </Popover.Content>
+      </>
+    ),
+  },
 } as Meta<Args>;
 
-type Args = React.ComponentProps<typeof PopoverExample>;
+type Args = React.ComponentProps<typeof Popover>;
+
+export const Default: StoryObj<Args> = {
+  play: async ({ canvasElement }) => {
+    // We want to test visual regression for the Popover.Content as well as the button,
+    // but don't want the drawer open initally outside Chromatic.
+    if (isChromatic()) {
+      const canvas = within(canvasElement);
+      const filtersTrigger = await canvas.findByRole('button');
+      filtersTrigger.click();
+    }
+  },
+};
+
+export const Arrow: StoryObj<Args> = {
+  args: {
+    children: (
+      <>
+        <Popover.Button as={Button}>Open Popover</Popover.Button>
+        <Popover.Content showArrow>
+          <div style={{ width: '250px' }}>
+            My popover content Lorem ipsum dolor sit amet, consectetur
+            adipiscing elit, sed do eiusmod
+          </div>
+        </Popover.Content>
+      </>
+    ),
+  },
+  ...Default,
```

Comment:
This is mostly for the play function, which doesn't work as a part of the `export default` block. If I should just copy paste the play function for more intuitiveness, lmk

Reply:
I like it this way 👍